app/forms/schools/add_participant_form.rb

```diff
@@ -108,26 +108,19 @@ def current_user
     end
 
     def save!
-      if type == :ect
-        EarlyCareerTeachers::Create.call(
-          full_name: full_name,
-          email: email,
-          cohort_id: school_cohort.cohort_id,
-          school_id: school_cohort.school_id,
-          mentor_profile_id: mentor&.mentor_profile&.id,
-        )
-      else
-        ActiveRecord::Base.transaction do
-          # TODO: What if email matches but with different name?
-          user = User.find_or_create_by!(full_name: full_name, email: email)
-
-          MentorProfile.create!(
-            user: user,
-            school_id: school_cohort.school_id,
-            cohort_id: school_cohort.cohort_id,
-          )
-        end
-      end
+      creator = case type
+                when :ect
+                  EarlyCareerTeachers::Create
+                when :mentor
+                  Mentors::Create
+                end
+      creator.constantize.call(
```

Comment:
Why do we need `constantize`? Internet tells me that it is a method on a string, that lets you find the class from that string, and use it as a class. 

But I don't think `creator` is a string, I think it is a class. So we could (I hope) just use `creator.call`. Am I thinking right?

Reply:
Changed.